src/code/z_demo.c

```diff
@@ -1,5 +1,6 @@
 #pragma increment_block_number "gc-eu:128 gc-eu-mq:128 gc-jp:128 gc-jp-ce:128 gc-jp-mq:128 gc-us:128 gc-us-mq:128" \
                                "ique-cn:128 ntsc-1.0:0 ntsc-1.1:0 ntsc-1.2:0 pal-1.0:0 pal-1.1:0"
+#include "padmgr.h"
```

Comment:
is it needed? or is it a leftover? no worries either way it's just an include but I'm curious